notebooks/harmony subsetting/Harmony L2 Subsetter.ipynb

```diff
@@ -46,7 +46,7 @@
   },
```

Comment:
<p>I don't understand what's going on here... why are we getting no data for both lat and lon? That doesn't seem like a "pass" condition to me?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/podaac/tutorials/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>if a granule ends up with a global bounding box (possibly due to a FORGE error), there is a chance our search "finds" it but it doesn't actually contain data in specified bounding box. This is a false positive, and we err on the side of including it so we always give a suer more data than they might actually need, not less than they want. It's not ideal, but it happens enough that 'failing' here is not too helpful either.</p>